es/criteria/version-control-and-history.md

```diff
@@ -35,20 +35,19 @@ El control de versiones distribuido te permite:
 * Garantizar que el código se ejecute correctamente.
 * Garantizar los colaboradores.
 
-## Cómo testear
+## Cómo probar
```

Comment:
I would like to keep the word "Testear". It is in the RAE dictionary (https://dle.rae.es/testear?m=form) and also refers directly to the word 'test', as in perform software checks. Perhaps we can use "hacer pruebas o tests"